pyk/src/pyk/kcfg/kcfg.py

```diff
@@ -1306,7 +1321,8 @@ def read_node_data(cfg_dir: Path, node_id: int) -> KCFG.Node:
         return KCFG.Node.from_dict(store.read_node_data(node_id))
 
 
-class KCFGExtendResult(ABC): ...
+class KCFGExtendResult(ABC):
+    info: str = field(default='')
```

Comment:
Consider

```suggestion
    info: str
```

and adding the `info: str = field(default='')` to each `dataclass` subclass.

Alternatively, do not make it an abstract attribute, since the value is not used for most subclasses.

Reply:
Would it be ok to just have it in the ones where it's needed? The compilation does not seem to complain.

Reply:
Yes, but then it shouldn't be present in the abstract class.

Reply:
Yes, I removed it from there.